lib/axios.ts

```diff
@@ -6,8 +6,9 @@ export const http = axios.create({
 });
 
 http.interceptors.request.use(async (config) => {
-	const accessToken =
-		"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNzE2OTEwNTQzLCJpYXQiOjE3MTUxODI1NDMsImp0aSI6ImNlMDFhZTQ0MTM1YzQ4OTNiNGEyZGZlODQ4MjcxNzJlIiwidXNlcl9pZCI6MTl9.N9VLh0gnlSavHl-REGr7pWFvl3UEKkT-c24pj8QG_F8";
+	const userItem = localStorage.getItem("user");
+	const userData = userItem ? JSON.parse(userItem) : {};
+	const accessToken = userData.access || "";
```

Comment:
Why not cookies? How can we use these tokens in the server?

Reply:
![image](https://github.com/Hospital-Database/Frontend/assets/81932663/88422b46-4fa2-401a-9ccc-e14f74e625df)

Reply:
- I have an idea, we can use server action with client request to set `user` object in the `localStorage` and `session` and make a separate `http` for each one.
- but for now, we can skip it to save time

Reply:
How you get statistics in the server?

Reply:
I got it in the client

Reply:
Ok it is in the client. We can skip it for now, but it won't take time though.